lib/feature_flagger/control.rb

```diff
@@ -13,15 +13,27 @@ def released?(feature_key, resource_id)
     end
 
     def release(feature_key, resource_id)
-      @storage.add(feature_key, resource_id)
+      resource_name = FeatureFlagger::Storage::RedisKeys.extract_resource_name_from_feature_key(
```

Comment:
Should calls made to `RedisKeys` only be made inside the Redis Storage adapter?

Or is it something we would be able to re-use to other types of storages? In that case, I think we should rename RedisKeys to something else.